OpenRA.Mods.Common/Widgets/Logic/Lobby/SpawnSelectorTooltipLogic.cs

```diff
@@ -92,7 +92,7 @@ public SpawnSelectorTooltipLogic(Widget widget, ModData modData, TooltipContaine
 			flag.IsVisible = () => playerFaction != null;
 			flag.GetImageCollection = () => "flags";
 			flag.GetImageName = () => playerFaction;
-			team.GetText = () => teamMessage.Update(playerTeam);
+			team.GetText = () => playerTeam > 0 ? teamMessage.Update(playerTeam) : "";
```

Comment:
This should not be visible when `playerTeam == 0`?

Reply:
Do we use `string.Empty`? I have nothing against it but I just have don't remember ever seeing it

Reply:
It doesn't make much of a difference in practice, so we can stay with `""`.